lib/src/list.dart

```diff
@@ -199,7 +199,7 @@ class ListNotificationsController<T extends Object> extends NotificationsControl
   }
 
   @override
-  void onChanges(Handle changesHandle) {
+  void onChanges(HandleBase changesHandle) {
```

Comment:
```suggestion
  void onChanges(RealmHandle changesHandle) {
```

Reply:
Will rename accordingly.

Reply:
Actually we already have a `RealmHandle` type. What alternative do you prefer - `CoreHandle`, `CapiHandle`, `RealmHandleBase`, or something else?

Reply:
I suggest we name these `RealmHandle` to better reflect the difference with Dart `Handle`

Reply:
argh...